tests/test_functions.py

```diff
@@ -527,3 +527,28 @@ def test_fstrings3(x):
         tmp = f"{x.__class__.__name__} foo"
         if tmp.startswith("Tensor"):
             return x + 1
+
+    # # This is to test the new syntax for pattern matching
+    # # ("match ... case ...") added on python 3.10.
+    # # Uncomment these test cases if you run on 3.10+
+    # @make_test
+    # def test_match_sequence(a):
```

Comment:
As Python(<3.10) doesn't support the ```match...case...``` syntax, so there are compile errors if we uncomment it when running py3.7-3.9 tests. Let me know if you have better solution. @jansel

Reply:
Let's just leave this commented out for now. I haven't seen a lot of users of this new syntax yet.